echo/server/dembrane/transcribe.py

```
@@ -57,7 +57,7 @@ def queue_transcribe_audio_runpod(
             input_payload["language"] = language
 
         data = {"input": input_payload}
-        logger.debug(f"***data: {data}")
+        logger.debug(f"data: {data}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)